packages/relay/src/lib/eth.ts

```diff
@@ -256,6 +257,57 @@ export class EthImpl implements Eth {
     return EthImpl.zeroHex;
   }
 
+  /**
+   * Always returns NO_MINING_WORK error.
+   */
+  getWork(): JsonRpcError {
+    this.logger.trace('getWork()');
+    return Errors['NO_MINING_WORK'];
```

Comment:
```suggestion
    return Errors['NO_MINING_WORK'];
```
```suggestion
    return predefined.NO_MINING_WORK
```

Reply:
I think this should just be UNSUPPORTED_METHOD